content/applications/marketing/marketing_automation/advanced/understanding_metrics.rst

```diff
@@ -2,79 +2,84 @@
 Understanding metrics
 =====================
 
-Metrics are detailed statistics that measure the progress (and success) of marketing campaigns.
-When creating marketing activities in a workflow, Odoo will visually display various metrics
-related to the campaign with graphs and data.
+Metrics are detailed statistics that measure the progress (and success) of marketing campaigns. When
+creating marketing activities in a workflow, Odoo visually displays various metrics related to the
+campaign with detailed graphs and in-depth data.
 
 Consider the following example:
 
 .. image:: understanding_metrics/commercial-prospection-campaign-example.png
    :align: center
    :alt: An example of a marketing automation campaign in Odoo.
 
-In this case, the :guilabel:`Target` of this campaign is set to :guilabel:`Lead/Opportunity` and
-was narrowed down to only focus on the Leads (or Opportunities) whose :guilabel:`Tag Name` contain
-a value of `product` and they *also* have an email address set up in the database.
+In this case, the :guilabel:`Target` of this campaign is set to :guilabel:`Lead/Opportunity`, and
+was narrowed down to *only* focus on the leads (or opportunities) whose :guilabel:`Tag Name`
+contains a value of `product` - and *also* have an email address set up in the database.
```

Comment:
think an em dash is more appropriate here as hyphens are used to connect words together, not clauses in a sentence. 

Note: GitHub doesn't appear to really show the difference visually between the two so here's [more info](https://www.thesaurus.com/e/grammar/em-dash/).
```suggestion
contains a value of `product`— and *also* have an email address set up in the database.
```